Scoop/Controller/TripDetailsViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  TripDetailsViewController.swift
+//  Scoop
+//
+//  Created by Richie Sun on 2/22/23.
+//
+
+import UIKit
+
+class TripDetailsViewController: UIViewController {
+    
+    var currentRide: Ride?
+    
+    private let stackView = UIStackView()
+    private let creatorLabel = UILabel()
+    private let creatorProfile = UIImageView()
+    private let creatorEmail = UILabel()
+    private let transportationMethod = UILabel()
+    private let driverType = UILabel()
+    private let locations = UILabel()
+    private let arrivalLocation = UILabel()
+    private let departureLocation = UILabel()
+    private let departureDateLabel = UILabel()
+    private let departureDate = UILabel()
+    private let numberTravelersLabel = UILabel()
+    private let numberTravelers = UILabel()
```

Comment:
Why two numberTravelersLabel?

Reply:
One is the actual label, the other displays the numbers